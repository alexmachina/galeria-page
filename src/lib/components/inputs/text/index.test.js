import React from "react";
import TextInput from "./";
import renderer from "react-test-renderer";

describe("Components", () => {
  describe("Inputs", () => {
    describe("Text", () => {
      it("Renders correctly", () => {
        const tree = renderer
          .create(
            <TextInput
              label="Full Name"
              value="Javier Escuella"
              onChange={a => a}
            />
          )
          .toJSON();

        expect(tree).toMatchSnapshot();
      });
    });
  });
});
