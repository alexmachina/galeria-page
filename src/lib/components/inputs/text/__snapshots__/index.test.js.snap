// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components Inputs Text Renders correctly 1`] = `
<div
  className="root-0-2-1"
>
  <div
    className="label-0-2-2"
  >
    <span
      className="root-0-2-8"
    >
      Full Name
    </span>
  </div>
  <div
    className="inputContainer-0-2-3"
  >
    <input
      className="input-0-2-4"
      onChange={[Function]}
      type="text"
      value="Javier Escuella"
    />
  </div>
  <div
    className="errorMessage-0-2-7"
  />
</div>
`;
